data_juicer/utils/mm_utils.py

```diff
@@ -786,3 +786,13 @@ def parse_string_to_roi(roi_string, roi_type='pixel'):
             '"[x1, y1, x2, y2]".')
         return None
     return None
+
+
+def close_video(container):
+    """
+    Close the video stream and container to avoid memory leak.
+
+    :param container: the video container.
+    """
+    container.streams.video[0].close(strict=False)
```

Comment:
Not an issue, but I wander that will there be a video[1]?

Reply:
a loop has been added here